installer_fabfile.py

```diff
@@ -780,3 +783,176 @@ def uninstall(dropdb=True, dropuser=True):
 
     with fab_settings(warn_only=True):
         local('yum remove -y goldstone-server')
+
+
+def get_ip():
+    import socket
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    try:
+        # doesn't even have to be reachable
+        s.connect(('10.255.255.255', 0))
+        ip = s.getsockname()[0]
+    except:
+        ip = '127.0.0.1'
+    finally:
+        s.close()
+    return ip
+
+
+def configure_notification_driver(filename):
+    """Configure a notification_driver setting for ceilometer."""
+    from fabric.contrib.files import contains, sed
+
+    with fab_settings(warn_only=True, user="root"):
+        # Set up syslog shipping
+        print(green("\nConfiguring notification_driver settings for " +
+                    filename))
+
+        has_messagingv2 = contains(
+            filename,
+            '^notification_driver[\s]*=.*messagingv2', escape=False)
+
+        has_empty_driver = contains(
+            filename, '^notification_driver[ ]*=[ ]*$', escape=False)
+
+        if not has_messagingv2:
+            if has_empty_driver:
+                print(green("\nReplacing empty notification_driver entry"))
+                sed(filename, '^notification_driver[ ]*=[ ]*$',
+                    'notification_driver=messagingv2')
+            else:
+                print(green("\nAdding a new notification_driver entry in "
+                            "DEFAULT section"))
+                sed(filename, '^\[DEFAULT\][\s]*$',
+                    '\[DEFAULT\]\\nnotification_driver=messagingv2')
+        else:
+            print(green("\nnotification_driver already configured"))
+
+
+@task
+def configure_stack(goldstone_addr=None, restart_services=None, accept=False):
+    """Configures syslog and ceilometer parameters on OpenStack hosts.
+
+    :param goldstone_addr: Goldstone server's hostname or IP accessible to
+                         OpenStack hosts
+    :type goldstone_addr: str
+    """
+
+    from fabric.contrib.files import upload_template
+
+    if not accept:
+        accepted = prompt(cyan("This utility will modify configuration files "
+                               "on the hosts supplied via the -H parameter, "
+                               "and optionally restart OpenStack and syslog "
+                               "services.  Do you want to continue (yes/no)?"),
+                          default='no', validate='yes|no')
```

Comment:
This prompt is functionally correct, and the text is understandable.

But it's displayed one long string that wraps at the window's edge.  I don't know if prompt() text can contain \n, but if it can, I suggest you insert a couple of \n characters so that the prompt text is less than ~ 30 characters wide.

This is what I see now:

![screen shot 2015-06-11 at 1 08 58 pm](https://cloud.githubusercontent.com/assets/231148/8117169/107eacd8-103b-11e5-9fb5-4517e341d046.png)